modules/gdscript/gdscript_tokenizer.cpp

```diff
@@ -294,7 +294,11 @@ void GDScriptTokenizerText::push_expression_indented_block() {
 
 void GDScriptTokenizerText::pop_expression_indented_block() {
 	ERR_FAIL_COND(indent_stack_stack.is_empty());
-	indent_stack = indent_stack_stack.back()->get();
+
+	if (!_is_at_end()) {
+		indent_stack = indent_stack_stack.back()->get();
+	}
+
```

Comment:
Probably the same change should be made to `GDScriptTokenizerBuffer`.

https://github.com/godotengine/godot/blob/89001f91d21ebd08b59756841426f540b154b97d/modules/gdscript/gdscript_tokenizer_buffer.cpp#L411-L415

Reply:
Yes, I think about that. But I was a little bit confused by the fact that `GDScriptTokenizerBuffer` has no `_is_at_end()` method. And right now I don't know how to write tests for this class. I'll look into it and add to PR if needed.

Reply:
@dalexeev so I added a `.bin.gd` test and also generated a `.gdc` file and separately tested the `GDScriptTokenizerBuffer` (via `GDScriptParser`). In summary, the error is not reproducible in `GDScriptTokenizerBuffer`. All is well.